site/en/workflows.md

```diff
@@ -1007,12 +1114,36 @@ accessible format that can be used as-is by expert and layman alike.
       </pre></code>
       </td>
       <td>&nbsp;</td>
-      <td><code>ocrd-pagetopdf -I PAGE-FILGRP -O PDF-FILEGRP -p '{"textequiv_level" : "word"}'</code></td>
+      <td><code>ocrd-pagetopdf -I PAGE-FILEGRP -O PDF-FILEGRP -p '{"textequiv_level" : "word"}'</code></td>
     </tr>
 
 </tbody>
 </table>
 
+### Step 20: Dummy Processing
```

Comment:
As mentioned above, not sure if this warrants an extra section. If you do make this a topic, please also explain that this is not merely a verbatim copy, but the PAGE-XML gets **re-generated** from the current namespace schema/model.